src/riak_ql_parser.yrl

```diff
@@ -1210,7 +1213,7 @@ assert_desc_key_field_type(DDL, ?SQL_PARAM{ name = [Name] }) ->
         _ ->
             return_error_flat(
                 "Elements in the local key marked descending (DESC) must be of "
-                "type sint64 or varchar, but was ~p.", [Type])
+                "an integer or binary type, but was ~p.", [Type])
```

Comment:
I prefer the old error message where it used keywords instead of abstracted words, the user can copy the type directly from the error message. I don't think we define what is an integer or binary type anywhere. The original message is wrong by that standard, it should include `timestamp`!